core/src/main/scala/org/apache/spark/Accumulator.scala

```diff
@@ -26,7 +26,7 @@ package org.apache.spark
  *
  * An accumulator is created from an initial value `v` by calling
  * [[SparkContext#accumulator SparkContext.accumulator]].
- * Tasks running on the cluster can then add to it using the [[Accumulable#+= +=]] operator.
+ * Tasks running on the cluster can then add to it using the `+=` operator.
```

Comment:
I just decided to keep original format rather than trying to make this pretty.

The original was as below:

- Scala
  <img width="140" alt="2016-11-26 12 46 33" src="https://cloud.githubusercontent.com/assets/6477701/20637823/6f1c8914-b3d6-11e6-83f4-87355205d4c1.png">

- Java
  <img width="121" alt="2016-11-26 12 46 46" src="https://cloud.githubusercontent.com/assets/6477701/20637824/6f1cfce6-b3d6-11e6-93d7-2bae071f5753.png">